docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command:
+      sh -c "python manage.py wait_for_db &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
```

Comment:
The `depends_on` directive is used correctly to ensure that the `app` service waits for the `db` service to be ready. However, note that `depends_on` does not wait for `db` to be 'ready' (i.e., accepting connections), it only waits for the container to start. The `wait_for_db` command in your application handles the readiness check.